src/phoenix/server/main.py

```diff
@@ -126,6 +126,7 @@ def _load_items(
     atexit.register(_remove_pid_file)
 
     parser = ArgumentParser()
+    parser.add_argument("--database", required=False)
```

Comment:
I think a more verbose flag would be helpful, e.g., `--database-url` or `--database-connection-url`.

Reply:
I'll do database-url. I struggled with how complicated I wanted this to be but true, It's more clear that it's a string